amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2091,7 +2094,148 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/events':
+    get:
+      tags:
+        - events
+      summary: List all the record management events
+      description: |
+        Use this to get all the events in the record management site. This return all the custom events as well as default events in record management site with mandatory fields and properties
+
+      operationId: getAllEvents
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/EventsInfo'
```

Comment:
Should this be `EventsInfoPaged`?  I think you are missing the standard paging parameters too.